equinix/resource_network_device.go

```diff
@@ -635,6 +743,70 @@ func createNetworkDeviceUserKeySchema() map[string]*schema.Schema {
 	}
 }
 
+func createClusterNodeDetailRequestSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		neDeviceClusterSchemaNames["LicenseFileId"]: {
+			Type:        schema.TypeString,
+			Optional:    true,
+			ForceNew:    true,
+			Sensitive:   true,
+			Description: neDeviceClusterDescriptions["LicenseFileId"],
+		},
+		neDeviceClusterSchemaNames["LicenseToken"]: {
+			Type:        schema.TypeString,
+			Optional:    true,
+			ForceNew:    true,
+			Sensitive:   true,
+			Description: neDeviceClusterDescriptions["LicenseToken"],
+		},
+		neDeviceClusterSchemaNames["VendorConfiguration"]: {
+			Type:     schema.TypeMap,
+			Optional: true,
+			ForceNew: true,
+			Elem: &schema.Schema{
+				Type:         schema.TypeString,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+			Description: neDeviceClusterDescriptions["VendorConfiguration"],
+		},
+	}
+}
+
+func createNetworkDeviceClusterNodeSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		neDeviceClusterSchemaNames["UUID"]: {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: neDeviceClusterDescriptions["UUID"],
+		},
+		neDeviceClusterSchemaNames["Name"]: {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: neDeviceClusterDescriptions["Name"],
+		},
+		neDeviceClusterSchemaNames["Node"]: {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: neDeviceClusterDescriptions["Node"],
+		},
+		neDeviceClusterSchemaNames["AdminPassword"]: {
```

Comment:
Isn't `AdminPassword` part of the `VendorConfiguration` block?

Reply:
Thanks for pointing that out. I have removed that from our terraform schema. It looks like we have API duplication for this field. Will fix that in parallel.